.github/workflows/ci.yml

```
@@ -144,3 +144,11 @@ jobs:
           path: ./
           exclude-dirs: cbmc
           include-file-types: .c,.h,.dox
+  doxygen:
+    runs-on: ubuntu-20.04
```

We can use `ubuntu-latest` instead as GitHub had transitioned to use Ubuntu 20.04 in the beginning of the year
```suggestion
    runs-on: ubuntu-latest
```